test/core/unit/task_launcher_test.rb

```diff
@@ -11,7 +11,6 @@ class TaskLauncherTest < ActiveSupport::TestCase
         let(:launcher_input) { { 'action_class' => Support::DummyDynflowAction.to_s, 'action_input' => input } }
         let(:input) { { :do => :something } }
         let(:expected_result) { input.merge(:callback_host => {}) }
-        let(:expected_batch_result) { expected_result.merge('callback' => nil) }
```

Comment:
@adamruzicka I guess since we're sending the results from each action vs. the parent one, we don't expect the callback to be nil, right?